src/raven.js

```diff
@@ -133,6 +134,12 @@ var Raven = {
     install: function() {
         if (isSetup() && !isRavenInstalled) {
             TraceKit.report.subscribe(handleStackInfo);
+
+            // Install all of the plugins
+            each(plugins, function(_, plugin) {
```

Comment:
This would just be a local, and we're not using underscore or lodash inside this loop, so should be fine.